### PR TITLE
Noam/feature/update default suites

### PR DESCRIPTION
**Pull request by noamzbr**
Update the default suites

---
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/deepchecks/deepchecks/blob/main/CONTRIBUTING.rst

### CODE REVIEW

File: deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ class PropertySegmentsPerformance(WeakSegmentsAbstractText):
     def __init__(self,
                  properties: Union[Hashable, List[Hashable], None] = None,
                  ignore_properties: Union[Hashable, List[Hashable], None] = None,
-                 n_top_properties: Optional[int] = 10,
+                 n_top_properties: Optional[int] = 15,
```
Comment (Nadav-Barak) [inline deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py:158]: may be slow

File: deepchecks/utils/dataframes.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 
 def default_fill_na_per_column_type(df: pd.DataFrame, cat_features: t.Union[pd.Series, t.List]) -> pd.DataFrame:
     """Fill NaN values per column type."""
+    pd.set_option('mode.chained_assignment', None)
```
Comment (Nadav-Barak) [inline deepchecks/utils/dataframes.py:30]: maybe put this in the general deepchecks init

**Review by Nadav-Barak — APPROVED** [review]

File: deepchecks/utils/dataframes.py (hunk 2 above)
Comment (noamzbr) [reply deepchecks/utils/dataframes.py:30]: I prefer to know about these things, so prefer not bo block categorically

File: deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py (hunk 1 above)
Comment (noamzbr) [reply deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py:158]: For now prefer this extra slowness to the warning and missing properties...

**Pull request merged**